test/elm/string/string-test.ts

```diff
@@ -177,6 +177,8 @@ describe('Length', () => {
   });
 
   it.skip('should be a Length', function () {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
What's going on here with these two?

Reply:
Great question. I think I got confused here because the error was that "Length doesn't exist on type str", where `str` is all the exports from `src/elm/string.ts`. There's a comment in that file that says "Length is completely handled by overloaded#Length" and I think I was like uhhhhh pass.

After looking at it further, Length is clearly defined in `src/elm/overloaded.ts`, which makes me confused as to why the test was accessing `str.Length` instead of importing `overloaded` and accessing `overloaded.Length` instead. After actually using my brain, perhaps it's because this test is `it.skipped`, and Length _used_ to be in `string.ts` but no longer is.

I think the actual solution is to remove these comments and import `Length` from `overloaded.ts` and update the assertion. Thoughts?

Reply:
must've lost steam/brainpower at this point in the conversion lol

Reply:
Was using overloaded.js what was being used before? Perhaps that is the right thing to do?

Also it is a skipped test. We could revisit this later.

Reply:
Eh I feel pretty confident that the class just moved from string.js to overloaded.js

I'm gonna update the import since that seems logical, and leave the test as skipped. We can always revisit down the line, but I think this is more logical than a TS ignore